.vscode/settings.json

```diff
@@ -1,16 +1,19 @@
 {
     "editor.formatOnSave": false,
-    "eslint.autoFixOnSave": true,
```

Comment:
Any reason this was removed?

Reply:
Deprecated in favor of `editor.codeActionsOnSave`, which is already enabled for eslint.